Source/MQTTnet/Client/Options/MqttClientOptionsBuilderTlsParameters.cs

```diff
@@ -27,7 +27,11 @@ public class MqttClientOptionsBuilderTlsParameters
         public IEnumerable<X509Certificate> Certificates { get; set; }
 #endif
 
-        public bool AllowUntrustedCertificates { get; set; }
+        #if NETCOREAPP3_1
```

Comment:
Please fix all the formatting issues (intention, missing empty lines, missing spaces in get;set etc.).